PdfJsViewerPlugin.inc.php

```diff
@@ -1,29 +1,30 @@
 <?php
 
 /**
- * @file plugins/generic/pdfJsViewer/PdfJsViewerPlugin.inc.php
+ * @file plugins/generic/pdfJsViewer/pdfJsViewerPlugin.inc.php
```

Comment:
Note that the filename here still has a capital `P`. Our class naming conventions are that names should start with a capital letter.